interfaces/HO-Portal/src/assets/i18n/en.json

```diff
@@ -254,7 +254,7 @@
         },
         "timestamp": "Latest import has been done on: {{dateTime}}",
         "btn-text": "Import People Affected",
-        "explanation": "The required format of the CSV-file is: <pre class='ion-color-primary'>phoneNumber,namePartnerOrganization\n+0000000000,Example-Name</pre>",
+        "explanation": "The required format of the CSV-file is: <pre class='ion-color-primary'>phoneNumber,namePartnerOrganization,paymentAmountMultiplier\n+0000000000,Example-Name,1</pre>",
```

Comment:
I think we need to move the actual file-format/syntax to some other place in the code as well..

That way we can ensure it will stay 'in sync' with the actual functionality..

Reply:
can you make this change?

Reply:
I'll make a separate PR 👍